src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './Books/bookSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
+});
```

Comment:
- [ ] Kindly make sure to add the `categoriesReducer` in your `configureStore` method here just like you did for the `booksReducer`.

Reply:
Thanks will do it now